Makefile

```diff
@@ -99,16 +98,20 @@ check-go-format: prebuild-check
 
 .PHONY: golint
```

Comment:
If it is an analysis, why not make it less confusing and use `analyze-go-code` as the Makefile target instead?